scilpy/tracking/local_tracking_branching.py

```diff
@@ -0,0 +1,396 @@
+# -*- coding: utf-8 -*-
```

Comment:
Looks like this file is an edited copy of `scilpy/tracking/local_tracking` where branching is assumed by default. Either we undo changes to `scilpy/tracking/local_tracking` and use only this file for branchings, or we keep changes to `local_tracking.py` and delete this current file. I don't know what is better, but in any case we do not need both.